packages/mui-utils/src/getScrollbarSize/getScrollbarSize.ts

```diff
@@ -1,7 +1,7 @@
 // A change of the browser zoom change the scrollbar size.
 // Credit https://github.com/twbs/bootstrap/blob/488fd8afc535ca3a6ad4dc581f5e89217b6a36ac/js/src/util/scrollbar.js#L14-L18
-export default function getScrollbarSize(doc: Document): number {
+export default function getScrollbarSize(doc: Document, win: Window = window): number {
   // https://developer.mozilla.org/en-US/docs/Web/API/Window/innerWidth#usage_notes
   const documentWidth = doc.documentElement.clientWidth;
-  return Math.abs(window.innerWidth - documentWidth);
+  return Math.abs(win.innerWidth - documentWidth);
```

Comment:
Would this ever be any different than?
```suggestion
  const win = doc.defaultView || window;
  return Math.abs(win.innerWidth - documentWidth);
```

Reply:
That does seem like it would behave the same way. And it would by extension fix all other usages of getScrollbarSize. If using the window of the given document is the intended behavior, then your suggested change is definitely better. I will change to that

Reply:
> I also added the || window since doc.defaultView might be null.

We'd be comparing document width to a window it is not associated with. I feel like it would make more sense to error in that case 🤔.

I feel like it almost would make more sense to start from the `window` instead of the `document`

```tsx
export default function getScrollbarSize(win: Window = window): number {
  // https://developer.mozilla.org/en-US/docs/Web/API/Window/innerWidth#usage_notes
  const documentWidth = win.document.documentElement.clientWidth;
  return Math.abs(win.innerWidth - documentWidth);
}
```